import { AxiosError } from 'axios';

export const errorInterceptor = (error: AxiosError) => {
  if (error.message === 'Network Error') {
    return Promise.reject(new Error('Erro de conex√£o.'));
  }

  if (error.response?.status === 401) {
    // TODO: se usar rota com login, implementar esse tratamento
  }

  return Promise.reject(error);
};
